analyzers/rspec/vbnet/S2692_vb.net.json

```
@@ -0,0 +1,16 @@
+{
+  "title": "\"indexOf\" checks should not be for positive numbers",
```

We need to change rule title VB.NET in Jira and regenerate this. The subtask title should look similar to the C# one.
```suggestion
  "title": "\"IndexOf\" checks should not be for positive numbers",
```